tests/test_coolest_slsim_interface.py

```diff
@@ -0,0 +1,47 @@
+from slsim.Util.coolest_slsim_interface import (
+    update_coolest_from_lenstronomy_slsim,
+    create_lenstronomy_slsim_from_coolest,
+)
+import pickle
+import os
+
+
+def test_update_coolest_from_lenstronomy_slsim():
```

Comment:
can you make a test going both ways first generating a coolest file, then reading it into a slsim object and then back, and compare whether the back and forth results in the same outcome?